lib/php/lib/Thrift/Protocol/TBinaryProtocolAccelerated.php

```diff
@@ -62,4 +62,9 @@ public function isStrictWrite()
   {
     return $this->strictWrite_;
   }
+
+  public function isBinaryAccelerated()
+  {
+    return true;
+  }
```

Comment:
I agree we need something like this but the name can be more general.
For example, `isAccelerated` or even `isDynamic`.
(Thinks about the day when we add TCompactProtocolAccelerated or anything without generated read/write calls...)